homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -20,36 +23,50 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         add_entities(devices)
 
 
-class MaxCubeShutter(BinarySensorEntity):
-    """Representation of a MAX! Cube Binary Sensor device."""
+class MaxCubeBinarySensorBase(BinarySensorEntity):
+    """Base class for maxcube binary sensors."""
 
     def __init__(self, handler, device):
         """Initialize MAX! Cube BinarySensorEntity."""
-        room = handler.cube.room_by_id(device.room_id)
-        self._name = f"{room.name} {device.name}"
         self._cubehandle = handler
         self._device = device
+        self._room = handler.cube.room_by_id(device.room_id)
+        self._attr_entity_category = ENTITY_CATEGORY_DIAGNOSTIC
```

Comment:
This should be a class variable

Reply:
Changed https://github.com/home-assistant/core/pull/58699/commits/4e328371499388fe6cb4db64bf746f788c797297